app/views/shared/schools/mentors/_can_not_remove.html.erb

```diff
@@ -0,0 +1,11 @@
+<%# locals: (mentor:, school:, support: false) -%>
+<label class="govuk-label govuk-label--l">
+  <span class="govuk-caption-l">
+    <%= support ? "#{mentor.full_name} - #{school.name}" : mentor.full_name %>
+  </span>
+  <%= t(".title") %>
+</label>
+
+<p class="govuk-body">
+  <%= simple_format(t(".description", mentor_name: mentor.full_name)) %>
+</p>
```

Comment:
This looks like a Placements-specific partial.

Reply:
It looks like the `app/views/shared` directory is currently being used in a couple of different ways:

1. for partials which are shared between both the School Placements and Funding Mentors services
2. for partials which are shared between 'regular' user journeys and 'support' user journeys in the support console

In the case of the latter, it does mean partials in the 'shared' directory could end up being service specific. For example, the following partials are Claims-specific:

- [app/views/shared/organisations/_grant_conditions.html.erb](https://github.com/DFE-Digital/itt-mentor-services/blob/7bd4c4d926ab679b0690bcf6a5a9298357f56541/app/views/shared/organisations/_grant_conditions.html.erb)
- [app/views/shared/organisations/_grant_funding.html.erb](https://github.com/DFE-Digital/itt-mentor-services/blob/7bd4c4d926ab679b0690bcf6a5a9298357f56541/app/views/shared/organisations/_grant_funding.html.erb)

I don't consider this to be a blocker, but if you feel strongly that the directory needs tidying up then feel free to record it as tech debt on the [shared dev board](https://trello.com/b/o9tcUSwc/itt-mentor-services-dev-tasks).